ui/cryptomaterial/tab_nav.go

```diff
@@ -0,0 +1,85 @@
+package cryptomaterial
+
+import (
+	"image"
+
+	"gioui.org/layout"
+	"gioui.org/op/clip"
+	"gioui.org/op/paint"
+
+	"github.com/crypto-power/cryptopower/ui/values"
+)
+
+type TabNavigation struct {
+	list          *ClickableList
```

Comment:
Why can't you reuse the existing component `NewTabItem`? Add your unique UI upgrades to that component.

Reply:
The reason is the NewTabItem codebase seems deprecated and will need a major clean up job.
Adding code along side the deprecated codes, does not work well for me plus the lines of codes in that tabs.go file is already over 300.